php-practice/section3.php

```diff
@@ -83,12 +83,12 @@
 $emailAddress = array_pop($usersToSendEmail);
 
 // On the next 2 lines, var_dump $emailAddress and print_r $usersToSendEmail.
-
-
+var_dump($emailAddress);
+print_r($usersToSendEmail);
 
 // What do they contain and why?
-// $emailAddress contains: [ANSWER HERE]
-// $usersToSendEmail contains: [ANSWER HERE]
+// $emailAddress contains: ['josie@example.com']
+// $usersToSendEmail contains: ['joe@example.com','joseph@example.com', 'joey@example.com', 'jose@example.com',]
```

Comment:
Since both of your answers kind of look like arrays, I just wanted to clarify here: `$emailAddress` is going to contain a string. (`'josie@example.com'`, not `['josie@example.com']`)

I think you already got that, but just wanted to make sure. Let me know if you have any questions about why.